dev-python/mongomock/mongomock-3.19.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+#DISTUTILS_USE_SETUPTOOLS=bdepend
+PYTHON_COMPAT=( python3_{6,8} )
+DISTUTILS_IN_SOURCE_BUILD=1
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="Library to help testing Python code that interacts with MongoDB via Pymongo."
+HOMEPAGE=" https://github.com/mongomock/mongomock https://pypi.org/project/mongomock/"
+SRC_URI="https://github.com/mongomock/mongomock/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="test" # it has a lot of network tests
```

Comment:
you can selectively disable the failing tests using `pytest --deselect path/file.py::test`

Reply:
The tests are mixed in. Found out I could set `NO_LOCAL_MONGO=1` to disable them.